meta-lmp-base/classes/lmp-disable-gplv3.bbclass

```diff
@@ -6,4 +6,5 @@ PACKAGECONFIG:remove:pn-curl = "libidn"
 PACKAGECONFIG:remove:pn-bluez5 = "readline mesh"
 PACKAGECONFIG:append:pn-bluez5 = " libedit "
 PACKAGECONFIG:remove:pn-iproute2 = "elf"
+PACKAGECONFIG:remove:pn-wireguard-tools = "wgquick"
```

Comment:
Still missing the patch that adds the pkgconfig logic into wireguard-tools, right?

Reply:
Yes, it's on the way